src/Microsoft.AspNetCore.Mvc.Core/MvcOptions.cs

```diff
@@ -127,5 +127,11 @@ public int MaxModelValidationErrors
         /// Gets a list of <see cref="IValueProviderFactory"/> used by this application.
         /// </summary>
         public IList<IValueProviderFactory> ValueProviderFactories { get; }
+
+        /// <summary>
+        /// Gets or sets the SSL port that is used by this application. If not set
+        /// the port won't be specified in secured url, e.g., https://localhost/path.
```

Comment:
- "URL" in all caps
- Suggest removing the commas.
- Suggest mentioning `[RequireHttps]` because nothing else uses this value. Otherwise customers may go searching for "secured URL".